admin/scss/_menu.scss

```diff
@@ -106,7 +108,8 @@
 		position: relative !important;
 		top:auto !important;
 		left:auto !important;
-		box-shadow: inset #C1C7CC -1px 0 0
+		box-shadow: inset #C1C7CC -1px 0 0;
+		min-height: calc(100% - 104px);
```

Comment:
I've added a variable in another merge request ( https://github.com/silverstripe/silverstripe-framework/pull/5317 ) which has a variable to use 2 * $navbar-total-height